lib/gpu/lib/src/render_pass.dart

```diff
@@ -214,6 +220,9 @@ base class RenderPass extends NativeFieldWrapperClass1 {
       if (error != null) {
         throw Exception(error);
       }
+
+      final texture = renderTarget.depthStencilAttachment!.texture;
+      _setScissor(0, 0, texture.width, texture.height);
```

Comment:
While at least one attachment must always be present, the DepthStencilAttachment is optional.
I'd recommend not working out what the default stencil should be here and just leave the default to Impeller.